### ISSUE TITLE
Socket Serving panic

### ISSUE THREAD

**Issue by MindHunter86** [MindHunter86/aniliSeeder#16]
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xcc8f33]

goroutine 66 [running]:
github.com/MindHunter86/aniliSeeder/deluge.(*Client).GetTorrents(0x0)
	/workspaces/repos/aniliSeeder/deluge/torrent.go:28 +0x73
github.com/MindHunter86/aniliSeeder/app.(*cmds).statCurrentTorrents(0x1072747)
	/workspaces/repos/aniliSeeder/app/cmds.go:48 +0x585
github.com/MindHunter86/aniliSeeder/app.(*SockServer).runClientCmd(0xc00013c378, 0x2)
	/workspaces/repos/aniliSeeder/app/socket.go:150 +0x1b8
github.com/MindHunter86/aniliSeeder/app.(*SockServer).clientRpcHandler(0xc00013c378, {0x1317ac8, 0xc00035a000})
	/workspaces/repos/aniliSeeder/app/socket.go:99 +0xd52
created by github.com/MindHunter86/aniliSeeder/app.(*SockServer).Serve
	/workspaces/repos/aniliSeeder/app/socket.go:56 +0x574
```

todo:
- disable socket serving for worker
- use grpc in socket commands

**Comment by MindHunter86** [comment]
https://github.com/MindHunter86/aniliSeeder/blob/d8f9b5cb35ec37b6468990bf0a6b9e6995dfafdb/app/app.go#L87

**Comment by MindHunter86** [comment]
fixed in 0c90c8299d90c07be3c1d3dcdc84a46c90693561
